### ISSUE TITLE
Support :db/retract without value

### ISSUE THREAD

**Issue by tonsky** [tonsky/datascript#339]
Same what rectractAttribute does now https://forum.datomic.com/t/datomic-0-9-6045-now-available/1360

**Comment by ont-app** [comment]
Would this be a suitable test?
```
(deftest test-retract-without-value-339
  (let [db (-> (d/empty-db {:aka    { :db/cardinality :db.cardinality/many }
                            :friend { :db/valueType :db.type/ref }})
               (d/db-with [ { :db/id 1, :name  "Ivan", :age 15, :aka ["X" "Y" "Z"], :friend 2 }
                           { :db/id 2, :name  "Petr", :age 37 } ]))]
    (testing "Retract without providing v"
      (let [db (d/db-with db [[:db/retract 1 :name]])]
        (is (= (d/q '[:find ?e ?n
                      :where [?e :name ?n]]
                    db)
               #{[2 "Petr"]}))
        (is (= (d/q '[:find ?a ?v
                      :where [1 ?a ?v]]
                    db)
               #{[:friend 2] [:age 15] [:aka "Z"] [:aka "Y"] [:aka "X"]}))))))
```

**Comment by tonsky** [comment]
Yes! I would suggest leaving second check only. And add another test case where `:aka` attribute is removed (it is multi-arity, better check that too)

**Comment by ericdscott** [comment]
How's this?
https://github.com/tonsky/datascript/pull/340

**Comment by tonsky** [comment]
Fixed in `0ac6cdf746a21c06d5dcbd4dc94463311df06ba8` 0.18.10
